### PR TITLE
Redesigned Login UI

### PR DESCRIPTION
**Pull request by hickoryeyes**
- Redesigned the login page with Xamarin Android assets only.
- You'll notice that some objects will now be automatically recolored to fit the theme.

**Some thoughts below...**
- Xamarin Android optimizes layouts for each device, but the automation makes it messy on devices with different screen density. Separate .xaml layouts would fix the problem ([source](https://docs.microsoft.com/en-us/xamarin/android/app-fundamentals/resources-in-android/resources-for-varying-screens?tabs=windows)). I think we can leave device support for later and work with the normal/most common option, unless anyone in the team has a different device.
- I suggest we move to Xamarin Forms for more functionality (access to more plugins and tutorials) and possible future iOS support, which is possible with Xcode ([source](https://docs.microsoft.com/en-us/xamarin/ios/get-started/installation/device-provisioning/free-provisioning?tabs=windows))

### CODE REVIEW

**Review by MigleMic — APPROVED** [review]
Looks great!
I also agree that we should consider migrating to xamarin.forms because we could add iOS and also there are some methods are useful that can be only accessed when using xamarin.forms.

**Review by Waltzerizm — APPROVED** [review]

File: Encounter1/Resources/values/styles.xml [hunk 1]
```diff
@@ -1,11 +1,18 @@
 <resources>
 
-    <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
-        <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
-    </style>
+	<style name="EncounterMe.LightMode" parent="@style/Theme.AppCompat.NoActionBar">
```
Comment (Mdominykas) [inline Encounter1/Resources/values/styles.xml:3]: For me location list became too light (white letters on grey screen)
https://pasteboard.co/6RzIUrTgvuMU.png

File: Encounter1/Resources/values/styles.xml [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+		<item name="android:colorBackground">@color/encounter_light_background</item>
+		<item name="android:windowBackground">@color/encounter_light_background</item>
+		<item name="android:colorAccent">@color/encounter_accent_1</item>
+		<item name="android:colorPrimary">@color/encounter_light_text</item>
```
Comment (Mdominykas) [inline Encounter1/Resources/values/styles.xml:7]: Same problem with menu and username as with locationList.
whiteMenu

Comment (hickoryeyes) [reply Encounter1/Resources/values/styles.xml:7]: Yes, it's like that on purpose. I haven't applied styling properties to them yet.

Comment (hickoryeyes) [reply Encounter1/Resources/values/styles.xml:7]: Rewrote the code a bit, so the text is dark once again :)

**Pull request merged**